website/pages/docs/drivers/docker.mdx

```diff
@@ -784,6 +790,10 @@ plugin "docker" {
     sources. The script's name must begin with `docker-credential-` and this
     option should include only the basename of the script, not the path.
 
+    If you set an auth helper, it will be tried for all images, including
+    public images. If you mix private and public images, you will need to
+    include `auth_soft_fail=true` in every job using a public image.
```

Comment:
might be a bit recursive, but linking back to `auth_soft_fail` may be nice

Reply:
Done in f610b11